app/controllers/games_controller.rb

```diff
@@ -39,4 +35,12 @@ def update
   def game_params
     params.require(:game).permit(:name)
   end
+
+  def flash_notices
```

Comment:
I really like the use of this flash_notices method to keep the rest of the codebase dry in regards to flash messages. Good job.